### DISCUSSION TITLE
First Class RedHat Support

### DISCUSSION THREAD

**Discussion by f5-ericmarden** [Ulauncher/Ulauncher#971] [Packaging]
Redhat now has free developer licenses. It would be great if you provided first-class RHEL8+ support, either as install instructions or with a RHEL specific repository.

**Comment by friday** [comment]
Thank you for the suggestion :+1:

Moved to https://github.com/Ulauncher/Ulauncher/discussions/categories/packaging

It's a lot of work to add support for a additional distros, reading documentation etc, so this is not something I am able to handle with the time I have at least. Plus, we have been looking into replacing the entire system we have for releasing packages now.
